products/health-care/beneficiary-travel/engineering/data-flow-sequence-diagram.md

```diff
@@ -0,0 +1,33 @@
+The basic sequence for most BTSSS is straightforward:
+
+```mermaid
+sequenceDiagram
+autonumber
+actor veteran as Veteran
```

Comment:
It might be good to add a simplified version of the authentication sequence here. I wouldn't go into detail, but enough to show there's some authN happening. Maybe a link to the SiS authN sequence diagram: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Sign-In%20Service/auth_flows/web_oauth.md.

Reply:
They updated the SiS docs and broke that link, but I believe[ this it the same diagram](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Products/Sign-In%20Service/auth_flows/cookie_oauth.md#sequence-diagram)?

Reply:
Agreed, let's add a note or some type of reference to the diagram.

Reply:
updated in 236c079de4e0976d37505b4cdefb32c454b3ed4c